inst/app/server.R

```diff
@@ -686,7 +687,7 @@ if(is.null(input$regfunctions)){
     }
     
     ## updateCheckboxInput
-    for (i in c("dirichlet", "rangeRestrict")) {
+    for (i in c("dirichlet", "imputeMissing", "rangeRestrict")) {
```

Comment:
I added the new input to the list of inputs that must be updated if one loads a model that was downloaded before.